<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObserMultiple</class>
 <widget class="QWidget" name="ObserMultiple">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="w_ctrl" native="true">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_addCanvas">
            <property name="text">
             <string>增加画布</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_removeCanvas">
            <property name="text">
             <string>减少画布</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_showSingle">
            <property name="text">
             <string>唯一</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_export">
            <property name="text">
             <string>导出数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_import">
            <property name="text">
             <string>导入数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_contiune">
            <property name="text">
             <string>持续缓存</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_run">
            <property name="text">
             <string>启动</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="le_maxNum">
            <property name="accessibleName">
             <string/>
            </property>
            <property name="accessibleDescription">
             <string/>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string>40</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>max number</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>32</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="pb_injectionData">
            <property name="text">
             <string>注入测试数据</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pb_clear">
            <property name="text">
             <string>全部清除</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_clearSingle">
            <property name="text">
             <string>选中清除</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QTreeWidget" name="tw_select">
       <column>
        <property name="text">
         <string>观测通道</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="w_multiPlot" native="true">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
